packages/sdk/src/Actions/EventAction.ts

```diff
@@ -66,17 +73,19 @@ export type {
   PrimitiveType,
 };
 
-type ReadEventActionParams<
+export type EventLogs = GetLogsReturnType<AbiEvent, AbiEvent[], true>;
```

Comment:
took me a sec to figure out why but it is technically fewer generic args than the `Log` alternative to get to the strict